apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -217,6 +231,8 @@ export class SendMessagePush extends SendMessageBase {
         content,
         payload,
         overrides,
+        subscriber,
+        step,
```

Comment:
this data is needed only for the `webhook` provider, so I'm sending them as a separate fields